src/components/Legend/Flex.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const Flex = styled.div`
```

Comment:
What is a `Flex`? I think the name is not really explanatory, even when thinking about flexbox. When do I use this Component and why does it set some borders?